src/main/java/com/contrastsecurity/resource/ConfigResource.java

```diff
@@ -82,7 +82,7 @@ public Response testConnection(final TeamServerProfile profile, @Context HttpSer
 			return Response.status(Status.UNAUTHORIZED).build();
 		}
 		try {
-			ContrastSDK contrastsdk = new ContrastSDK(profile.getUsername(), profile.getApikey(), profile.getServicekey(), profile.getUrl());
```

Comment:
Were these always in the wrong order?

Reply:
Yes, and for some reason in the VulnerabilityTypes file it is in the correct order.